### PR TITLE
HW16 done

### PR DESCRIPTION
**Pull request by ArtemBerdnik**

### CODE REVIEW

File: HW16/pageobjects/CartPage.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+from selenium.webdriver.support.wait import WebDriverWait
+
+from HW16.pageobjects.BasePage import BasePage
+from HW16.utils.PropertiesHandler import get_properties
+from selenium.webdriver.remote.webdriver import WebDriver
+
+LOGGER = logging.getLogger()
+
+
+class CartPage(BasePage):
```
Comment (AnastasiiaGor) [inline HW16/pageobjects/CartPage.py:13]: Page object should be a representation of the page and shouldn't contain asserting methods

File: HW16/tests/test_hw16.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+from HW16.pageobjects.MainPage import MainPage
+
+
+def test_login(browser):
+    MainPage(browser)\
```
Comment (AnastasiiaGor) [inline HW16/tests/test_hw16.py]: test should contain separate steps and asserts to maintain tests and reports readability

**Review by AnastasiiaGor — CHANGES_REQUESTED** [review]

File: HW16/tests/test_hw16.py (hunk 2 above)
Comment (ArtemBerdnik) [reply HW16/tests/test_hw16.py]: reworked that part. 
Thanks

File: HW16/pageobjects/CartPage.py (hunk 1 above)
Comment (ArtemBerdnik) [reply HW16/pageobjects/CartPage.py:13]: Agreed that it is an open question  :) 

For now left it as is

**Review by AnastasiiaGor — APPROVED** [review]

**Pull request merged**